src/client/containers/BoardPage/ResultButton/ResultButtonPostDeadline.container.js

```diff
@@ -14,7 +14,7 @@ export default function ResultButtonPostDeadline() {
 
   return (
     <div className="Result">
-      <Link to={`/resultpage/${boardId}`}>
+      <Link to={`/boards/${boardId}/results`}>
```

Comment:
Try it out please and see if it takes you to the result page

Reply:
I'm waiting for Rasim to resolve his conflicts on his branch and merge otherwise our page is not functioning to test the above

Reply:
I don't understand what you mean? You simply click this above to see if it takes you to the result page. It has nothing to do with anything else

Reply:
@senner007 Yes Niels, It goes to the result page
![image](https://user-images.githubusercontent.com/75580873/131500317-9994d851-383d-4381-9d7a-402d24e0142e.png)